### ISSUE TITLE
Latest verison errors while trying to add movies from both my current library and recommendation etc

### ISSUE THREAD

**Issue by jscottdouglas** [Radarr/Radarr#3557]
I've reinstalled radarr about 4 times, even killed my entire database and reinstall, i can not add movies or discover movies. 

19-6-9 02:14:27.7|Warn|StaticResourceMapper|File /usr/share/radarr/UI/Content/Images/favicon/manifest not found
19-6-9 02:14:36.6|Error|SkyHookProxy|Failed to discover movies for action recommendations!

[v0.2.0.1344] NzbDrone.Core.MetadataSource.RadarrAPI.RadarrAPIException: Error while calling api: Unprocessable Entity
Full error(s): Unprocessable Entity (422, RayId: b0849d1f-6765-49d4-bbba-078629ea9793), Details: Please add some movies before using our recommendation engine :)
  at NzbDrone.Core.MetadataSource.RadarrAPI.RadarrAPIClient.Execute[T] (NzbDrone.Common.Http.HttpRequest request) [0x00055] in <e94d2b84efc543538bf12577e9022618>:0 
  at NzbDrone.Core.MetadataSource.RadarrAPI.RadarrAPIClient.DiscoverMovies (System.String action, System.Func`2[T,TResult] enhanceRequest) [0x00038] in <e94d2b84efc543538bf12577e9022618>:0 
  at NzbDrone.Core.MetadataSource.SkyHook.SkyHookProxy.DiscoverNewMovies (System.String action) [0x000a2] in <e94d2b84efc543538bf12577e9022618>:0 

when i try to add a movie from my library it just claims it can't find any match for it... i don't see an error in the log.

any help getting to the bottom of this would be great as it is, it's un usable it looks like it's trying to connect somewhere but just times out.

Debian with latest mono.

**Comment by jscottdouglas** [comment]
[v0.2.0.1351] NzbDrone.Core.MetadataSource.RadarrAPI.RadarrAPIException: Error while calling api: Unprocessable Entity
Full error(s): Unprocessable Entity (422, RayId: ad7013ff-f0bc-4d04-9e3d-8247d213a9fc), Details: Please add some movies before using our recommendation engine :)
  at NzbDrone.Core.MetadataSource.RadarrAPI.RadarrAPIClient.Execute[T] (NzbDrone.Common.Http.HttpRequest request) [0x00055] in <6ad6e1caebc14059a79f44cd8c8b145b>:0 
  at NzbDrone.Core.MetadataSource.RadarrAPI.RadarrAPIClient.DiscoverMovies (System.String action, System.Func`2[T,TResult] enhanceRequest) [0x00038] in <6ad6e1caebc14059a79f44cd8c8b145b>:0 
  at NzbDrone.Core.MetadataSource.SkyHook.SkyHookProxy.DiscoverNewMovies (System.String action) [0x000a2] in <6ad6e1caebc14059a79f44cd8c8b145b>:0

**Comment by IIIdefconIII** [comment]
I have the same problem, i just went over from package install on Synology to a docker version. For the docker I am using the latest tag.

I select the movies all folder
![image](https://user-images.githubusercontent.com/9917659/59156105-0bd8dd00-8a96-11e9-8b85-46b71512c1e0.png)

It finds 6 pages x 15

![image](https://user-images.githubusercontent.com/9917659/59156110-24e18e00-8a96-11e9-8039-df88a0899795.png)

I was able to up to 6 pages of 100 items,

whenever i went for 500 items, it stopped  at page 5 With The jurassic park as last title
While in fact there are more movies in this folder. Missing all items from the jurrassis park till movies starting with Z.

I had to add all found items and then search again in the movies/all folder to get the last movies from tje jurassic park till the end of the alphabet.

Looks like whenever you try adding more movies then (around) 500 its timesout and give an error that there a no movies.
